### PR TITLE
[ShapeUp] Renku V2 Project Page

### LINKED ISSUE #3098: [Project Page 2.0] Add Responsive Project Page Structure
To incorporate a project page for the new Renku 2.0, we need to implement the project page structure. This structure should include responsiveness for all screen sizes and incorporate the new styles specific to Renku.

### PR DESCRIPTION
**Pull request by andre-code**
This PR adds a new project page for Renku v2. It includes:
- Project information overview (name, namespace, description, visibility, keywords)
- Add session launcher from custom image or existing global environment
- Sessions launcher overview
- Launch, stop, modify, pause a session
- Add data source (cloud storage only)
- Data sources overview
- Link existing code repository
- Code repositories overview 
- Edit project information
- New colors for inputs, buttons, alerts, and fonts

fix #3098

/deploy renku=release-0.52.x

### CODE REVIEW

File: client/src/features/sessionsV2/components/SessionStatus/SessionStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+    () => NotebooksHelper.cleanAnnotations(annotations) as NotebookAnnotations,
+    [annotations]
+  );
+  const state = status.state;
+  const defaultImage = cleanAnnotations.default_image_used;
+
+  const badge =
+    state === "running" && defaultImage ? (
+      <SessionBadge className="border border-warning bg-warning-subtle">
+        <ExclamationCircleFill className="text-warning" size={16} />
```
Comment (leafty) [inline client/src/features/sessionsV2/components/SessionStatus/SessionStatus.tsx:85]: For all bootstrap icons, use class `"bi"` to align them with the surrounding text. Also, you can handle the margin here:

```suggestion
        <ExclamationCircleFill className={cx("bi", "me-1", "text-warning")}  size={16} />
```

File: client/src/features/sessionsV2/components/SessionStatus/SessionStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+      </SessionBadge>
+    ) : state === "running" ? (
+      <SessionBadge className="border border-success bg-success-subtle">
+        <CheckCircleFill className="text-success" size={16} />
+        <span className="text-success-emphasis">Running Session</span>
+      </SessionBadge>
+    ) : state === "starting" ? (
+      <SessionBadge className="border border-warning bg-warning-subtle">
+        <Loader size={16} className="text-warning" inline />
+        <span className="text-warning ml-2">Starting Session</span>
```
Comment (leafty) [inline client/src/features/sessionsV2/components/SessionStatus/SessionStatus.tsx:96]: Note: class `ml-2` does not exist. Use `ms-1`, `me-1`, etc.

File: client/src/styles/icons/sessions.svg
Comment (leafty) [inline client/src/styles/icons/sessions.svg:1]: [design] same viewbox comment here

**Review by leafty — CHANGES_REQUESTED** [review]
This is mostly ready :clap: , some comments below.

The **[design]** items are non-blocking but need to be raised with design and fixed shortly.

**Comment by andre-code** [comment]
Thanks, @leafty. I've included your code suggestions in the latest commit.

File: client/src/components/buttons/Button.tsx [hunk 3]
```diff
@@ -80,7 +86,7 @@ function ButtonWithMenu(props: ButtonWithMenuProps) {
     <>
       <DropdownToggle
         data-cy="more-menu"
-        className={`${props.className} ${classes}`}
+        className={cx(props.className || "", classes, "rounded-end-pill")}
```
Comment (leafty) [inline client/src/components/buttons/Button.tsx:89]: why have `|| ""`?

```suggestion
        className={cx(props.className, classes, "rounded-end-pill")}
```

File: client/src/components/buttons/Button.tsx [hunk 4]
```diff
@@ -287,11 +293,78 @@ function RoundButtonGroup({ children }: { children: React.ReactNode[] }) {
...
+  text: string;
+  to: string;
+  tooltip: ReactNode;
+}) {
+  const ref = useRef(null);
+  return (
+    <>
+      <span ref={ref} className={buttonStyles.LinkUnderline}>
+        <Link className="text-decoration-none" to={to}>
+          {text} <ArrowRight />
```
Comment (leafty) [inline client/src/components/buttons/Button.tsx:313]: This probably looks better:

```suggestion
          {text}
          <ArrowRight className={cx("bi", "ms-1")} />
```

**Review by leafty — COMMENTED** [review]
Should be good after this.

**Review by leafty — APPROVED** [review]
LGTM :rocket: :clap:

**Review by leafty — APPROVED** [review]

**Pull request merged**